### PR TITLE
v2 peoplePicker & Search Bar back Button Press focus Issue Fix

### PR DESCRIPTION
**Pull request by Anubhvv**
### Problem 
The focus stays in the people picker and it does not go away when back navigation button is pressed.
### Root cause 
onKeyEvent() Modifier is interfering with the logic of back button press of Fluent textField 
### Fix
While implementing onKeyEvent callback, return true to stop propagation of this event. If you return false, the key event will be sent to the parent.

### Validations

(how the change was tested, including both manual and automated tests)

### ScreenRecording
|                                  Before                                           |

https://github.com/microsoft/fluentui-android/assets/68989156/22fe7a53-6cc7-490c-a592-79b858128272

https://github.com/microsoft/fluentui-android/assets/68989156/408c1cc9-1a07-4da9-ae39-a39795ff0053

------------------------------------------------------------

|                                  After                                            |

https://github.com/microsoft/fluentui-android/assets/68989156/552723b5-f4d9-474f-a7c7-c4b2b39dc67e

https://github.com/microsoft/fluentui-android/assets/68989156/44dcd9cd-3b7f-47c9-a024-03a8d6e99221

### Pull request checklist

This PR has considered:
- [ ] Light and Dark appearances
- [ ] Automated Tests
- [ ] Documentation and demo app examples
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and RTL layouts
- [ ] Size classes and window sizes (notched devices, multitasking, different window sizes, etc)

### CODE REVIEW

File: fluentui_peoplepicker/src/main/java/com/microsoft/fluentui/tokenized/peoplepicker/PeoplePicker.kt [hunk 1]
```diff
@@ -137,7 +137,7 @@ fun PeoplePicker(
                         onValueChange(queryText, selectedPeopleList)
                     }
                 }
-                true
+                false
```
[the remark this answers is not in the document]
Comment (praveen970) [reply fluentui_peoplepicker/src/main/java/com/microsoft/fluentui/tokenized/peoplepicker/PeoplePicker.kt:140]: i think here the issue "back" button press. Irrespective of people picker having chips or not, when "back" button is pressed it should switch the activity. The chips will be removed on "backspace" press not the "back" press. The fix here is applied to fix the "back" press and does not interfere with the "backspace" logic.

File: FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2PeoplePickerActivity.kt [hunk 2]
```diff
@@ -15,7 +15,6 @@ import androidx.compose.runtime.setValue
 import androidx.compose.ui.ExperimentalComposeUiApi
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.pluralStringResource
-import androidx.compose.ui.res.stringResource
```
Comment (praveen970) [inline FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2PeoplePickerActivity.kt:18]: please remove this file from the PR

Comment (Anubhvv) [reply FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2PeoplePickerActivity.kt:18]: Unused import, so it's just code a clean-up effort.

File: fluentui_peoplepicker/src/main/java/com/microsoft/fluentui/tokenized/peoplepicker/PeoplePicker.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (mishramayank1) [reply fluentui_peoplepicker/src/main/java/com/microsoft/fluentui/tokenized/peoplepicker/PeoplePicker.kt:137]: Correct, I mixed up backspace with backpress.

**Review by mishramayank1 — APPROVED** [review]

**Pull request merged**